<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chuan</class>
 <widget class="QWidget" name="Chuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chuan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: rgb(85, 255, 255);
  color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
  font: 75 9pt &quot;ADMUI3Lg&quot;;
  background-color: rgb(170, 85, 255);
  font: 8pt &quot;ADMUI3Lg&quot;;
  font: italic 9pt &quot;Adobe 黑体 Std R&quot;;
  background-color: rgb(85, 0, 255);
  background-color: rgb(85, 255, 0);
  background-color: rgb(0, 170, 255);
}
QComboBox{
background-color: rgb(170, 85, 127);
  border-color: rgb(0, 170, 255);
  background-color: rgb(85, 85, 255);
}

QPushButton{
background-color: rgb(170, 0, 127);

}

QGroupBox{
background-color: rgb(170, 170, 255);
}

QTextEdit{
background-color: rgb(255, 255, 255);
  color: rgb(0, 0, 0);

}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>511</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,2" columnminimumwidth="1,1">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>选择串口 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbPortName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>波特率  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbBaudRate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>数据位  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbDataBits"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>奇偶校验位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbParity">
          <property name="styleSheet">
           <string notr="true">QCombo Box{
background-color: rgb(85, 170, 255);


}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>停止位  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbStopBits"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>流控制  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_6"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,3,1">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>数据接收区</string>
        </property>
        <widget class="QTextEdit" name="teReceiveData">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>数据发送区</string>
        </property>
        <widget class="QTextEdit" name="teSendData">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>291</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnOpen">
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSend">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>发送数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>ip</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>port</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="ip"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="port"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ipbtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>480</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置ip和端口地址</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="macaddr"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>361</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> 设置终端地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="macbtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>390</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置终端地址</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>ipbtn</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>label_9</zorder>
  <zorder>macbtn</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
